docs/docs-content/troubleshooting/enterprise-install.md

```diff
@@ -77,3 +77,77 @@ following steps to restart the management pod.
    ```shell hideClipboard
    pod "mgmt-f7f97f4fd-lds69" deleted
    ```
+
+## Run Script to Make vSphere CNS Mapping Unique
+
+In Palette releases 4.4.8 and earlier, self-hosted instances of Palette in vSphere do not have globally unique cluster
+IDs. This can lead to issues if you have multiple instances of self-hosted Palette in your vSphere environment, and can
+lead to data loss and data overwrite issues during an upgrade as well.
+
+This issue is resolved in Palette releases starting with 4.4.14. However, if you have self-hosted instances of Palette
+in your vSphere environment, you should execute the following utility script to make the Cloud Native Storage (CNS)
+mapping unique for the associated PVC.
+
+### Debug Steps
+
+1. SSH to the VM where you can access your self-hosted Palette instance with `kubectl`.
+
+2. Issue the following command to download the utility script.
+
+   ```bash
+   curl --output csi-helper https://software.spectrocloud.com/tools/csi-helper/csi-helper
+   ```
+
+3. Adjust the permission of the script.
+
+   ```
+   chmod +x csi-helper
+   ```
+
+4. Download the kubeconfig file of your self-hosted Palette instance.
```

Comment:
Let's explain here that you can get the Kubeconfig for the self-hosted cluster from the System Console and by navigating to the **System Profiles**  and accessing the cluster details page.